test-map.cpp

```diff
@@ -207,21 +207,17 @@ class Test {
     MapStrObj * mss = new MapStrObj();
     Object** keys = mss->get_keys();
     Object** values = mss->get_values();
-    t_true(keys != nullptr);
```

Comment:
I think this is actually what we want.
```
    /**
     * Returns a new Object array of keys. This array will need to be freed in memory after use.
     * NOTE: If a map has no keys, it will return an empty Object**
     * @return new Object array of keys
     */
    virtual Object** get_keys();

    /**
     * Returns a new Object array of values. This array will need to be freed in memory after use.
     * NOTE: If a map has no values, it will return an empty Object**
     * @return new Object array of values
     */
    virtual Object** get_values(); 
```

Reply:
Decided that we'll let it be.